userguide/tutorials/reference-time-examples.adoc

```diff
@@ -0,0 +1,91 @@
+= Reference Time/Fixed Offset Timezone
+
+In this document, we look a little bit deeper into the concepts of reference time and fixed offset timezone.
+
+== Reference Time
+
+Each account in Kill Bill has a *reference time* associated with it. It can be specified at the time of creating the account. If not specified, it defaults to the account creation date time. Kill Bill uses the reference time to convert a `LocalDate` to a `DateTime` and vice-versa. So for example, if the invoice for an account is due on a particular date, the reference time is used to determine the time of the day at which the invoice will actually be generated. Note that the reference time is always considered to be in UTC.
+
+Let us take a look at a few examples of how the reference time influences invoice generation.
+
+=== Example 1
+
+* Set the time to `2023-01-01T11:00`.
+* Create an account with `referenceTime=2023-01-01T10:30`.
+* Create a subscription to a monthly plan - Invoice is generated immediately.
```

Comment:
Need to add the fact that subscription is created `immediately`, i.e. by passing a `null` billing date; maybe add a pointer to the doc.